substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/Range.java

```diff
@@ -87,7 +87,7 @@ public Range(StringTable stringTable, MethodEntry methodEntry, int lo, int hi, i
         }
         this.methodEntry = methodEntry;
         this.fullMethodName = isInline ? stringTable.uniqueDebugString(constructClassAndMethodName()) : stringTable.uniqueString(constructClassAndMethodName());
-        this.fullMethodNameWithParams = stringTable.uniqueString(constructClassAndMethodNameWithParams());
```

Comment:
So putting this into the string-table was unnecessasy?

Reply:
Yes. The string table serves two purposes

1) deduplicate strings that may occur more than once so as to reduce the amount of memory used during debug info generation
2) optionally demarcate a subset of those strings as needing to be generated into the strings section and indexed.

In this case we know that the full method name with params string will only occur in this one instance and only be used for log messages. So, insertion in the string table is wasted effort.